backend/resources/models.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+import datetime
+from django.db import models
+from accounts.models import User
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+# Create your models here.
+
+
+FILE_TYPE_CHOICES = [("IMAGE", "image"), ("ATTACHMENT", "attachment")]
+
+
+def resource_directory_path(instance, filename):
+    # file will be uploaded to MEDIA_ROOT/<year>/<month>/<filename>
+    now = datetime.datetime.now()
+    return f"resources/{now.year}/{now.month}/{instance.id}"
```

Comment:
why not take the convention like `upload_to="account/photos/%Y/%M", null=True, blank=True`